examples/memoryll.py

```diff
@@ -94,32 +94,32 @@ def mknod(self, req, parent, name, mode, rdev):
         self.reply_entry(req, entry)
 
     def open(self, req, ino, fi):
-        print 'open:', ino
+        print('open:', ino)
         self.reply_open(req, fi)
 
     def read(self, req, ino, size, off, fi):
-        print 'read:', ino, size, off
+        print('read:', ino, size, off)
         buf = self.data[ino][off:(off + size)]
         self.reply_buf(req, buf)
 
     def readdir(self, req, ino, size, off, fi):
-        print 'readdir:', ino
+        print('readdir:', ino)
         parent = self.parent[ino]
         entries = [('.', {'st_ino': ino, 'st_mode': S_IFDIR}),
             ('..', {'st_ino': parent, 'st_mode': S_IFDIR})]
-        for name, child in self.children[ino].items():
+        for name, child in list(self.children[ino].items()):
```

Comment:
D'oh! I noticed that before and thought the same thing, but then forgot to change it back. I'll fix that.